src/hotspot/share/opto/split_if.cpp

```diff
@@ -31,7 +31,7 @@
 
 //------------------------------split_thru_region------------------------------
 // Split Node 'n' through merge point.
-RegionNode* PhaseIdealLoop::split_thru_region(Node* n, Node* region) {
+RegionNode* PhaseIdealLoop::split_thru_region(Node* n, RegionNode* region) {
   assert(n->is_CFG(), "");
   assert(region->is_Region(), "");
```

Comment:
This assertion can also be removed or moved to L445 as sanity check.

Reply:
Right, as_Region() takes care of the assert anyway.